spec/spec_helper.rb

```diff
@@ -32,4 +33,25 @@
   config.order = :random
 
   Kernel.srand config.seed
+
+  config.before(:all) do
```

Comment:
you probably need this only for one test, is that correct? in this case

* i would move this to the corresponding test
* clean up after all tests which write to the tmp directory (otherwise your tests have side effects, and executed in a different order may yield different test results)